src/aws-manager.js

```diff
@@ -1370,6 +1370,20 @@ class AwsManager {
       }
     }
 
+    for (let request of this.__internalState) {
+      if (request.WorkerType === workerName) {
+        response.internalTrackedRequests.push({
```

Comment:
Two small things... Can we rename the variable to something other than response? To avoid perpetuating legacy...
Also I think you wanted to delete the to-do below...